README.md

```diff
@@ -6,9 +12,18 @@ docker-credential-helpers is a suite of programs to use native stores to keep Do
 
 Go to the [Releases](https://github.com/docker/docker-credential-helpers/releases) page and download the binary that works better for you. Put that binary in your `$PATH`, so Docker can find it.
 
-### Building from scratch
+## Building
+
+You can easily build the credential helpers using Docker:
+
+```shell
+# create builder
+$ docker buildx create --use
+# build credential helpers and output to ./bin/build
+$ docker buildx bake "https://github.com/docker/docker-credential-helpers.git"
```

Comment:
Wondering if people expect the build to be from remote source (more likely they would clone the repo?)

Perhaps we should mention both options, otherwise someone would clone the repo, and then build from the remote source 😬

Reply:
Ah yes good point, will add also how to do it from local source